addon/helpers/app-version.js

```diff
@@ -0,0 +1,38 @@
+import { helper } from '@ember/component/helper';
+
+import config from 'ember-get-config';
```

Comment:
I think we can avoid using `ember-get-config` by making this a class-based helper and resolving the config ourselves.